CONTRIBUTING.md

```diff
@@ -1,91 +1,176 @@
+
+```bash
+make coverage
+
+go tool cover -html=coverage.out # Open coverage status in your browser
+``` 
+
+#### **Repositories**
+Horusec has other repositories and you can check the README for each one of them: 
+
+- [**Charts**](https://github.com/ZupIT/charlescd/tree/main/circle-matcher)
+- [**Devkit**](https://github.com/ZupIT/horusec-devkit)
+- [**Engine**](https://github.com/ZupIT/horusec-engine)
+- [**Jenkins**](https://github.com/ZupIT/horusec-jenkins-sharedlib)
+- [**Operator**](https://github.com/ZupIT/horusec-operator)
+- [**Platform**](https://github.com/ZupIT/horusec-platform)
+- [**VSCode plugin**](https://github.com/ZupIT/horusec-vscode-plugin)
+- [**Kotlin**](https://github.com/ZupIT/horusec-tree-sitter-kotlin)
+- [**Vulnerabilities**](https://github.com/ZupIT/horusec-examples-vulnerabilities)
+
+### **First contribution**
+Contributing to a new feature is only allowed in the [**main repository**](https://github.com/ZupIT/horusec).
+
+Before contributing to this repository, please discuss the changes you wish to make via e-mail or [**forum**](https://forum.zup.com.br/c/en/horusec/14). 
+
+### **Add new feature, bug fixing or improvement**
+If you want to add an improvement, a new feature or bugfix, follow the steps to contribute: 
+
+**Step 1:** Make sure your branch is based on main;
+
+**Step 2:** When opening an issue, choose a template to answer the questions regarding what you want to contribute: 
+- [**Bug Report**](https://github.com/ZupIT/horusec/blob/main/.github/ISSUE_TEMPLATE/bug_report.md)
+- [**Feature request**](https://github.com/ZupIT/horusec/blob/main/.github/ISSUE_TEMPLATE/feature_request.md)
+- [**Improvement**](https://github.com/ZupIT/horusec/blob/main/.github/ISSUE_TEMPLATE/improvement.md)
+- [**Support request**](https://github.com/ZupIT/horusec/blob/main/.github/ISSUE_TEMPLATE/support_request.md)
+
+**Step 3:** Make your changes and open a GitHub pull request;
+
+**Step 4:** Make sure to write a title describing what you have done;
+
+**Step 5:** Fill in the template in the PR, here you need to write what you did and how the team can verify it; 
+
+**Step 6:** You must commit to comply with the DCO rules. It will need to be [**signed-off**](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt--s) and [**verified**](https://docs.github.com/en/github/authenticating-to-github/managing-commit-signature-verification/about-commit-signature-verification). Example: ` git commit -s --amend`.
+
+
+### **Pull Request's approval**
+Your pull request is approved when:
+- 2 code owners approve it.
+- Pass all GitHub actions checking process (lint, test, coverage, license, build, e2e, security, dco).
+
+### **After your pull request's approval**
+- If it is a bug fix, the team will perform the changes and there will be a new release.
+- If it is a feature, it will be in the next release. 
+
+## **Community**
+
+- Do you have any question about Horusec? Send to out [**mailing list**](horusec@zup.com.br).
```

Comment:
Link returning not found

Reply:
Do you guys use this email?

Reply:
The email is correct, the problem was only in the link but it has already been fixed.